resources/views/checkout/pages/credentials.blade.php

```diff
@@ -6,7 +6,7 @@
 
 @section('content')
     <div class="container">
-        <x-rapidez-ct::layout class="mt-8 sm:mt-14">
+        <x-rapidez-ct::layout class="mt-8 sm:mt-14" v-cloak>
```

Comment:
Does the progress bar cause layout shifts? I thought it didn't use JS

Reply:
So you want v-cloak on the separate items instead of the wrapper?

Reply:
Depends on what's shifting